docs/_frontend/src/models/FilterForm/FilterForm.js

```diff
@@ -1,7 +1,14 @@
 import { createElement as e } from 'react';
 import Select from '../Select';
 import './FilterForm.css';
-import { products, productDisplayName } from '../ModelItem/utils';
```

Comment:
Let's don't drop the mapping between old and new names. If we deploy without changes in the Elasticsearch index, the website will display old names.

Reply:
Done